tests/test_datetime_import.rs

```diff
@@ -1,14 +1,18 @@
 #![cfg(not(Py_LIMITED_API))]
 
 use pyo3::{prelude::*, types::PyDate};
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+static COUNTER: AtomicUsize = AtomicUsize::new(0);
 
 #[test]
 #[should_panic(expected = "module 'datetime' has no attribute 'datetime_CAPI'")]
 fn test_bad_datetime_module_panic() {
     // Create an empty temporary directory
     // with an empty "datetime" module which we'll put on the sys.path
     let tmpdir = std::env::temp_dir();
-    let tmpdir = tmpdir.join("pyo3_test_date_check");
+    let id = COUNTER.fetch_add(1, Ordering::SeqCst);
```

Comment:
The `COUNTER` seems like a solution to the property of `cargo stress` that it runs tests in a loop. How about using something like a random number or a uuid (will need to add to `dev-dependencies`, but I think that's ok)?

Reply:
We talked about this on a call and decided to look to look at adding a dev dependency on `tempfile`